docs/versions.rst

```diff
@@ -62,7 +62,7 @@ Version warning
 ---------------
 
 This is a banner that appears on the top of every page of your docs that aren't stable or latest.
-This banner has a text with a link redirecting the users to the latest version of your docs.
+This banner has a text with a link redirecting the users to the stable version of your docs.
```

Comment:
This may still be confusing. I'd like to say something like

```suggestion
This banner has a text with a link redirecting the users to the latest stable version of your docs.
```

but with a clearer English sentence.

Reply:
Mmm... does the banner links to `v0.x` or to `stable`?

Reply:
We only have one stable version, there isn't something like `latest stable`.

Reply:
I realized that and that's why my second question.